_episodes/41-code-review.md

```diff
@@ -245,106 +176,8 @@ The following diagram depicts the branches that you should have in the repositor
-   (make sure you do it outside your software repository's folder you have been working on so far).
-   This will create a copy of the repository locally on your machine
-   along with all of its (remote) branches.
-   ~~~
-   $ git clone <remote-repo-url>
-   $ cd <remote-repo-name>
-   ~~~
-   {: .language-bash}
-2. Check with the repository owner (your team member)
-   which feature (SR1.1.1 or SR1.2.1) they implemented in the
-   [previous exercise](/32-software-design/index.html#implement-requirements)
-   and what is the name of the branch they worked on.
-   Let's assume the name of the branch was `feature-x`
-   (you should amend the branch name for your case accordingly).
-3. Your task is to add tests for the code on `feature-x` branch.
-   You should do so on a separate branch called `feature-x-tests`,
-   which will branch off `feature-x`.
-   This is to enable you later on to create a pull request
-   from your `feature-x-tests` branch with your changes
-   that can then easily be reviewed and compared with `feature-x`
-   by the team member who created it.
-
-   To do so, branch off a new local branch `feature-x-tests` from the remote `feature-x` branch
-   (making sure you use the branch names that match your case).
-   Also note that, while we say "remote" branch `feature-x` -
-   you have actually obtained it locally on your machine when you cloned the remote repository.
-   ~~~
-   $ git checkout -b feature-x-tests origin/feature-x
-   ~~~
-   {: .language-bash}
-
-   You are now located in the new (local) `feature-x-tests` branch
-   and are ready to start adding your code.
-
-#### Step 3: Adding New Code
 
-> ## Exercise: Implement Tests for the New Feature
-> Look back at the
-> [solution requirements](/31-software-requirements/index.html#solution-requirements)
-> (SR1.1.1 or SR1.2.1)
-> for the feature that was implemented in your shared repository.
-> Implement tests against the appropriate specification in your local feature branch.
->
-> *Note: Try not to not fall into the trap of
-> writing the tests to test the existing code/implementation -
-> you should write the tests to make sure the code satisfies the requirements
-> regardless of the actual implementation.
-> You can treat the implementation as a
-> [black box](https://en.wikipedia.org/wiki/Black-box_testing) -
-> a typical approach to software testing -
-> as a way to make sure it is properly tested against its requirements
-> without introducing assumptions into the tests about its implementation.*
-{: .challenge}
-
-> ## Testing Based on Requirements
-> Tests should test functionality,
-> which stem from the software requirements,
-> rather than an implementation.
-> Tests can be seen as a reflection of those requirements -
-> checking if the requirements are satisfied.
-{: .callout}
-
-Remember to commit your new code to your branch `feature-x-tests`.
-
-~~~
-$ git add -A
-$ git commit -m "Added tests for feature-x."
-~~~
-{: .language-bash}
```

Comment:
As above - reader is no longer required to write some tests for someone else to review